tools/generator/Java.Interop.Tools.Generator.CodeGeneration/CodeGenerator.cs

```diff
@@ -476,7 +499,7 @@ public void WriteInterface (InterfaceGen @interface, string indent, GenerationIn
 		// For each interface, generate either an abstract method or an explicit implementation method.
 		public void WriteInterfaceAbstractMembers (InterfaceGen @interface, ClassGen gen, string indent)
 		{
-			foreach (Method m in @interface.Methods.Where (m => !m.IsInterfaceDefaultMethod && !m.IsStatic)) {
+			foreach (var m in @interface.Methods.Where (m => (!opt.SupportDefaultInterfaceMethods || !m.IsInterfaceDefaultMethod) && !m.IsStatic)) {
```

Comment:
I don't understand the addition of `!opt.SupportDefaultInterfaceMethods || ` here.  Why would `generator --lang-features=default-interface-methods` alter what statement does?

Reply:
It does not seem to have any effect, removed.